### ISSUE TITLE
Build OpenSearch 1.x branch nightly

### ISSUE THREAD

**Issue by dblock** [opensearch-project/opensearch-build#39]
**Is your feature request related to a problem? Please describe.**

We have a bunch of changes on OpenSearch 1.x branch towards 1.0 and plugins are starting to need that. Need an artifact to run integration tests.

**Describe the solution you'd like**

A .tar.gz build nightly off OpenSearch 1.x branch.

**Additional context**

Currently waiting on this, is https://github.com/opensearch-project/k-NN/pull/22

**Comment by mch2** [comment]
Closing as duplicate of #350
